paulshryock/release-bump
# [#7]: Get package version and help documentation from CLI

## Summary
From the CLI, the user should be able to check the package version with `-v`/`--version` or access help documentation with `-h`/`--help`.
## Changelog
### Added
- Get package version from CLI. [#7]
- Get package help documentation from CLI.
### Changed
- Better logging.
- Cancel Changelog bump if file does not exist.
## Testing
- Run `release-bump -h` and see help information logged
- Run `release-bump -v` and see package version logged
## Ticket
Closes #7
## Screenshot(s)
N/A
## Checklist
- [x] I have tested the code in this pull request.
- [x] I have documented any new features or changes in the Readme.
